native/common/jp_encoding.cpp

```diff
@@ -55,7 +55,7 @@ struct membuf : std::streambuf
 	{
 		this->setg(begin, begin, end);
 	}
-};
+} ;
```

Comment:
:+1:

Reply:
Yep, this is the wonders of autoformatting using netbeans as the editor on C++.  It has tied spacing for one common feature to spaces for something annoying like this.  If I tell it no spaces after a close brace, then it does something like remove all spaces in operator sequences.   One checkbox does both behaviors. Thus I had to accept bad things like this to make common case acceptable.

Reply:
just wondering which IDE would do such things :)

Reply:
Netbeans is not a very good C++ GUI but for JPype 0.8 where I have much of the backend code in Java I need to keep bookmarks set between the two code bases it is useful.  I just wish they would allow a drop in for a  more genera in formatter so that I don't have to make horrible decisions.